bpython/repl.py

```diff
@@ -56,6 +58,7 @@
 
 
 class RuntimeTimer(object):
+    """Calculate running time"""
```

Comment:
I like docstrings, maybe we can do one better:
`"""Records cumulative time spend in this context manager"""`